R/loocv.R

```diff
@@ -191,8 +191,14 @@ is.baggr_cv <- function(x) {
 #' Given multiple [loocv] outputs, calculate differences in their expected log
 #' predictive density.
 #'
-#' @param x An object of class `baggr_cv` or a list of such objects.
+#' @param x A list of `baggr_cv` objects, with a minimum of 2 objects required for comparison.
+#'          `baggr_cv` objects can be created via the `loocv` method. In instances where a list
```

Comment:
`loocv` is a function, not a method, no?

Reply:
Yes, my mistake. I'll correct this.

Reply:
Also, it's not a list of objects. It's several objects given as arguments. It should accept a list, but seemingly it doesn't

Just to be clear, I want to be able to do either `loo_compare(bg, bg2, bg3)` or `loo_compare(list(bg, bg2, bg3))`. And ideally be able to name them (see other comment below).

Secondly, super minor, but despite of what it says, it can't be a single model, e.g. `loo_compare(bg)` gives error `'loo_compare' requires at least two models.` so the documentation is not true.

Reply:
Got it! I'll fix this. 

Just to clarify, when you note that the documentation implies a single model, are you referring to how the documentation lists just one parameter `x` and the second parameter is provided under optional arguments `...`? If so, I can adjust the documentation to read as -
```
@param x The first in a series of `baggr_cv` objects passed as arguments, with a minimum of 2 arguments required for comparison. `baggr_cv` objects can be created via the [loocv] function. In instances where more than 2 arguments are passed, the first model will be compared sequentially to all other provided models.
@param y The second `baggr_cv` object for comparison. 
@param ... Additional objects of class "baggr_cv"
```

Reply:
Latest commit updates doc and adds names. I played around with passing list objects as well, but I think that might be a bit more than a quick fix. The issue appears to lie somewhere within the `UseMethod` function, though I didn't dive in too deep yet. Would you like me to tackle that as well?

Reply:
'useMethod': This is probably in "learning for self" category. Not essential, but will come in handy. See here: http://adv-r.had.co.nz/OO-essentials.html
However, you should not need anytjing that complicated to pass a list of named objects. Have you seen hwo it's done by `baggr_compare`? I think that one has what you need (although speaking from memory)

Reply:
(maybe this is resolved by another comment)

Reply:
To clarify, `UseMethod` only necessary if you want to pass the argument as a list, e.g.: `loo_compare(list(loo,loo2,loo3))`, which currently returns: `Error in UseMethod("loo_compare") : no applicable method for 'loo_compare' applied to an object of class "list"`. Simply passing a list of named objects as arguments like `baggr_compare`, e.g. `loo_compare("loo3"=loo3,"loo"=loo,"loo2"=loo2)` works without any error from `UseMethod`.